docs/amber_script.md

```diff
@@ -124,33 +130,30 @@ The following commands are all specified within the `PIPELINE` command. If you
 have multiple entry points for a given shader, you'd create multiple pipelines
 each with a different `ENTRY_POINT`.
 
-Bind the entry point to use for a given shader. The default entry point is main.
 
 ```
+  // Bind the entry point to use for a given shader. The default entry point is
+  // main.
   ENTRY_POINT <shader_name> <entry_point_name>
 ```
 
-Shaders can be added into pipelines with the `ATTACH` call. Shaders may be
-attached to multiple pipelines at the same time.
-
 ```
-  ATTACH <name_of_vertex_shader>
-  ATTACH <name_of_fragment_shader>
+  // Shaders can be added into pipelines with the `ATTACH` call. Shaders may be
+  // attached to multiple pipelines at the same time.
+  ATTACH <name_of_shader>
```

Comment:
Suggest this:
```
# Assuming the module only has one entry point for that stage, then use that entry point.
# The <shader-type> must match the type used to create the module
ATTACH <shader-type> <name_of_shader_module> 
# Explicitly name the entry point 
ATTACH <shader-stage> <name_of_shader_module> ENTRY_POINT myEntry
```

Reply:
See shader Cl, we can move discussion there.